src/crimson/os/seastore/extent_placement_manager.h

```diff
@@ -171,7 +171,15 @@ class SegmentedAllocator : public ExtentAllocator {
 
 class ExtentPlacementManager {
 public:
-  ExtentPlacementManager() = default;
+  ExtentPlacementManager() {
+    devices_by_id.resize(DEVICE_ID_MAX, nullptr);
```

Comment:
That is a suprise to me that DEVICE_ID_MAX is not the global maximum device-id value, because the other dependent values DEVICE_ID_NULL ... DEVICE_ID_ZERO are more approriate to be globally unique and addr-type independent.

Anyway, I'll switch to define and use the "real" maximum value to work around this issue.

Reply:
done